frontend/institution/descriptionInst/description_inst.css

```diff
@@ -0,0 +1,53 @@
+#textarea-edit-description{
+    height: 20em !important;
```

Comment:
`!important` is really necessary?

Reply:
Yes, the material library is overwriting this property